rails-snow-queen/app/assets/javascripts/map.js

```diff
@@ -68,13 +68,15 @@ class Map {
   }
 
   addListeners() {
+    document.addEventListener("DOMContentLoaded", $('#addressSubmitModal').modal("show"));
```

Comment:
I don't think this is actually a `Map` responsibility. Probably better (although not _great_) in `application.js` or even straight in the `quotes#new` view.